src/Bicep.Core.Samples/Files/LoopsIndexed_LF/main.sourcemap.json

```diff
@@ -2914,148 +2914,148 @@
           "TargetLine": 428
         },
         {
-          "SourceLine": 2,
-          "TargetLine": 434
+          "SourceLine": 0,
```

Comment:
The source map entries are equivalent to original version. It seems to be a side effect of this line: https://github.com/Azure/bicep/blob/208784d82926785cfd5d284b1079c1575b3671ab/src/Bicep.Core/Emit/PositionTrackingJsonTextWriter.cs#L178.